EZP-31784 - Autosave description (#73)
ezsystems/user-documentation: docs/site_organization/working_with_page.md

    Before publishing or saving the Page, edit its title and description.
    Switch to the Field view using the [Page Builder toolbar](#page-builder-toolbar).

    !!! note "Autosave"

>         To help you preserve your work eZ Platform comes with an Autosave feature.

Reviewer: ```suggestion
        To help you preserve your work eZ Platform autosaves your progress.
```
